src/doc_emit/html_embedded_styles.css

```diff
@@ -114,6 +114,9 @@ ul ol { margin-top: 0px; }
 
 /* Make the target distinct; helps when we're navigating to a function */
 a:target + * { background-color: #FFFF99; }
+@media (prefers-color-scheme: dark) {
```

Comment:
This should really be using CSS variables (and thus defined at the top of the file) like the rest of the colour scheme.

Reply:
Shows how much I know about CSS.